clients/python/game/models.py

```diff
@@ -24,9 +27,12 @@ def __str__(self):
 Tick: %d
 Map: %s
 Players:
+  %s
+Resources:
   %s""" % (self.tick,
            self.map,
-           "\n  ".join([str(player) for player in self.players])))
+           "\n  ".join([str(player) for player in self.players]),
```

Comment:
Why is there a join with `str(player)` there? Is this line supposed to be above resources?